src/console/Microsoft.Health.Fhir.Ingest.Console.Common/Extensions/FhirTransformationExtensions.cs

```diff
@@ -78,7 +79,7 @@ public static IServiceCollection AddFhirImportServices(this IServiceCollection s
 
             services.AddSingleton<IExceptionTelemetryProcessor, FhirExceptionTelemetryProcessor>();
             services.AddSingleton<MeasurementFhirImportOptions>();
-            MeasurementImportServiceExtensions.AddImportService(services, config);
+            services.AddSingleton<IImportService, MeasurementGroupFhirImportService>();
```

Comment:
So we are just swapping the MeasurementFhirImportService for the MeasurementGroupFhirImportService since the latter supports both, making the configuration redundant?

Reply:
yeah I figured since the new class supports both we don't really need the configuration. I think the configuration was added so that we could toggle off the new class while we were investigating the one issue that ended up being due to a break in the event hub sdk partition assignment code.